mmf/models/frcnn.py

```diff
@@ -0,0 +1,555 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
For the feature extraction script that you had added earlier you should replace the function calls in that with the functions you have added here. That will help clean up the code inside tools/scripts.

Reply:
We talked about this off github but want it to be here for future ref, the GeneralizedRCNN in tools is different from the one in this commit because it calculates many other values like obj_ids, attr_ids, bounding boxes, etc. So we cannot reference the one here in tools. Hopefully in a future PR we can bring the two together and have just one version.